index.html

```diff
@@ -1,9 +1,543 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
     <head>
+        <title>TNW</title>
```

Comment:
Let's remove the HTML validator error. I am using [this validator](https://validator.w3.org/#validate_by_upload)